components/03-modules/dashboard-form/_dashboard-form.scss

```diff
@@ -0,0 +1,37 @@
+$dashborad-form-margin: 30px !default;
+
+.dashboard-form {
+    &--hidden {
+        display: none;
+    }
+
+    &__title {
+        margin-bottom: $dashborad-form-margin;
+        padding-bottom: 10px;
+        border-bottom: 1px solid $gray;
+    }
+
+    &__input {
+        margin-bottom: $dashborad-form-margin;
+    }
+
+    &__fields {
+        margin-bottom: $dashborad-form-margin;
+    }
+
+    &__checkbox {
+        margin-bottom: $dashborad-form-margin;
+    }
+
+    &__content {
+        margin-bottom: $dashborad-form-margin;
+    }
+
+    &__title-form {
+        margin-bottom: $dashborad-form-margin;
+    }
```

Comment:
it is used inside hidden field? So margin is added for what reasons?

Reply:
I don't know if I understand. This margin is added to keep it visibly clear, and this item is hidden b/c it doesn't contain script to show it. It's in `account-info` view to avoid importing errors

Reply:
ok. clear. IMO it should contain script to show, b/c some can think that element is hidden for good ;)